### ISSUE TITLE
Groups documentation is incorrect, or addGroup expects the wrong argument keys

### ISSUE THREAD

**Issue by Jon-Biz** [sporritt/jsPlumb#522]
The documentation for addGroups says that it should expect a `container` key in its options. It actually expects the dom element to be keyed `el`. To wit:

```
jsPlumb.addGroup({
  el: SOME_DOM_ELEMENT
  id: 'dom_element_id
})
```

https://jsplumbtoolkit.com/community/doc/groups.html#adding

* Is the documentation out of date, or should the code expect the DOM element as the `container` key? Should I put up a PR?
* Is there any reason why it can't take a `id` string?

**Comment by Jon-Biz** [comment]
Edit: I may be using this wrong in some way, but I can't figure out how. 

I'm trying to create an object with a drag handle, essentially: there are UI elements that the user interacts with contained the draggable container. 

Now, while passing an`el` instead of a `container` to the `addGroups` method, I can get the container to behave (becomes `draggable` by default and `draggable` is toggleable with a flag, as per the documentation), but I can't set internal elements not to be draggable. 

I am passing in a `dragOptions` object, with a `filter` key, with 'a selector, or selectors, identifying elements that should not cause a drag to begin'. I can't seem to make this do the trick.

Here's what I'm passing in:

```
sPlumbr.addGroup({
      el: Dom Element
    , id: 'GROUPID'
    , dragOptions: {
        filter: $(`#CHILDID`)
      }
    })
```

I've tried setting child to a selector string ( `"'#CHILD_ID"` and `".CHILD_CLASS"`), but nothing can stop the draggable listener from preventing me from interacting with the child element, bar setting `draggable` to false.

Thanks!

**Comment by Jon-Biz** [comment]
I figure out the issue I was having - it seems that you have to specify any children of child components as well. 

Since I'm using third party react components, which are generating 'dom' elements on the fly in response to interactions, there's no way to attach a selector beforehand. However, by assigning a wrapper `div` and selecting all it's children (ie - '#DIV_ID *'), I've been able to get this to work.

I'm leaving this ticket open so we can fix the document errors, or change the code to use 'container' instead of 'el', whichever is intended.

**Comment by sporritt** [comment]
docs corrected in 2.1.2 released today (el, not container)
